data/transactions/logic/assembler.go

```diff
@@ -868,7 +871,7 @@ func asmDefault(ops *OpStream, spec *OpSpec, args []string) error {
 	}
 	err = ops.Out.WriteByte(spec.Opcode)
 	if err != nil {
-		return lineErr(ops.sourceLine, err)
+		return ops.addLine(err)
```

Comment:
I think `lineErr` is a better name than `addLine`. The latter assumes adding error as line (?) to somewhere rather than returning a new error object.

Reply:
The point is that it is just adds a line to an existing error.  I want to make clear that this is not producing the error, it's only adding the line information to the existing error.  I don't feel very strongly about it.  I considered withLine() and annotate() as possibilities as well.